modules/packages/Image.chpl

```diff
@@ -67,16 +67,17 @@ module Image {
   // set helper params for colors
   //
   @chpldoc.nodoc
-  param red = 0,        // names for referring to colors
-        green = 1,
-        blue = 2,
-        numColors = 3;
+  enum rgbColor {
+    red=0,
+    green=1,
+    blue=2,
+  }
 
   //
   // Verify that config-specified pixelType is appropriate for storing colors
   //
   if isIntegral(pixelType) {
-    if numColors*bitsPerColor > numBits(pixelType) then
+    if (rgbColor.size)*bitsPerColor > numBits(pixelType) then
```

Comment:
You could still define `param numColors = rgbColor.size;`

Reply:
I'm not sure that adds much and takes another identifier away from being used locally for some variable name.